src/derp.c

```diff
@@ -118,7 +143,7 @@ static void read_from_network_worker(void *arg) {
 		goto end;
 	}
 
-	ESP_LOGE(TAG, "Read from network worker starting");
+	ESP_LOGI(TAG, "Read from network worker starting");
```

Comment:
`ESP_LOGE(TAG, "Read from interface worker starting");` <- You need to change to INFO as well